language/tools/move-mv-llvm-compiler/tests/rbpf-tests/debug-print.move

```diff
@@ -1,13 +1,71 @@
 // log 10
+// log 11
+// log 12
+// log 13
+// log 14
+// log 15
+// log @00000000000000000000000000000000000000000000000000000000000000AB
+// log true
```

Comment:
I wonder what these lines (esp. long line at 7) stand for? Can you explain, please.

Reply:
> I wonder what these lines (esp. long line at 7) stand for? Can you explain, please.

See `move-mv-llvm-compiler$ less docs/Development.md` which explains `log` and the other directives.

Reply:
Also, the long hex number starting with `@` on line 7 is a Move address. See `https://move-language.github.io/move/address.html#address`. FWIW, I recommend reading the whole Move book.

Reply:
Ty, Jason.